### ISSUE TITLE
[Idea] Storing all code as a comment on the Mod-block

### ISSUE THREAD

**Issue by LennardF1989** [Ludonope/BFPortalUnleashed#4]
I just tested this and it seems the "Add Comment" option on a Mod-block is stored as an XML-element, making it perfect for storing all the code!

That way you can read back the code from a Mod-block without needing to use LocalStorage and it will always remember, even after clearing your browser cache.

![image](https://user-images.githubusercontent.com/869973/143787299-98f51e33-1f50-4963-996c-9eaba57d9386.png)

**Comment by Ludonope** [comment]
Oh great! I was also thinking maybe I can store actual comments in the code as comments on block along with pseudo-variable names to make debugging a bit easier when you start getting many variables.

**Comment by LennardF1989** [comment]
Yep! That is how I started, I wanted to look at the possibilities of making debugging easier, then I thought... why not just store code in there as well.
